<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundWidget</class>
 <widget class="QWidget" name="BackgroundWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lPosition">
     <property name="text">
      <string>Direction:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lEnabled">
     <property name="text">
      <string>Enabled:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="chkEnabled">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lType">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>13</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="KComboBox" name="cbType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lFileName">
     <property name="text">
      <string>File name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="leFileName">
     <property name="toolTip">
      <string>Specify the name of the image file.</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="bOpen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string> Select the file to import</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lColorStyle">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="KComboBox" name="cbColorStyle"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lImageStyle">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="KComboBox" name="cbImageStyle"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lBrushStyle">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="KComboBox" name="cbBrushStyle"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lFirstColor">
     <property name="text">
      <string>First color:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="KColorButton" name="kcbFirstColor"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="lSecondColor">
     <property name="text">
      <string>Second color:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="KColorButton" name="kcbSecondColor"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lOpacity">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QSpinBox" name="sbOpacity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
     </property>
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="KComboBox" name="cbPosition">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
